week-1/implement/sum.js

```diff
@@ -0,0 +1,18 @@
+function sumNumbers(arr) {
+    let result = 0;
+if(arr.length ==0){return 0;}
+    for (let i = 0; i < arr.length; i++) {
+        if (typeof arr[i] === 'number') {
+            result += arr[i];
+        }
+    }
+
```

Comment:
Nice work. But can you remove all logs below please?

Reply:
@Sabella-8 when you removed the logs, you can also remove `numbers`, `numbe`, and `numb` variables from the global scope, as they are not used now